src/transpiler/transpiler.ts

```diff
@@ -27,16 +27,18 @@ export async function transpileFiles(directory: string, outputDir: string, optio
          */
         const bundles = await rollup({
             input: files,
-            onwarn: ()=>{},
+            onwarn: () => {},
             plugins: [
                 babel({
                     cwd: ROOT_DIR,
                     babelHelpers: "bundled",
                     plugins: [
-                        "source-map-support"
+                        "source-map-support",
                     ],
                     presets: [
-                        "@babel/preset-env",
+                        ["@babel/preset-env", {
+                            targets: { node: "12.16" },
```

Comment:
Would this require all users of this library to at least use node v12.16?

Reply:
Yes, but it's only related to transpilation process, not to rendering, someone can still use `render` function without problems. This is hardcoded for generator, because it has also in requirements `12.16` as minimum version.

Reply:
Hmmm... Isn't it the other way around? you can transpile the files using another node version, but rendering the actual transpiled code would require node >v12.16 right?

Reply:
You can transpile but for what? 😄 At the moment we only transpile templates for generator, and without >= 12.16 node you cannot use generator - https://github.com/asyncapi/generator#requirements If someone will raise issue, we will make options for that :) The main problem is that `async await` from node 12 is natively supported and we don't need extra helpers inside bundled template. If you wanna go with older version you must handle this issue https://github.com/rollup/rollup/issues/3160 and trust me, I spent 2-3 hours on reading issues and no one has solution with bundled those helpers - some helpers must be included as imports and in our case this is a huge problem and it's not acceptable, because then every template must have some polyfilles installed as dependency :P

Reply:
Alright, just ensure the `readme` contains the updated requirement then 😄

Reply:
Added :)

Reply:
@magicmatatjahu did the transpilation work with >= v12 of node?

Reply:
@jonaslagoni  This is requirement only for running transpiled template.  Maybe main `Requirements` section is not a good idea. Maybe something like this? https://github.com/asyncapi/generator-react-sdk/pull/43/commits/d2a0401a09f7b52cce476a2789d2ca28d34bbfca

Reply:
Better yes, but did you say it worked for v12? No need to specify 12.16 if it works for the major version itself 😄

Reply:
I want to point out that it works with the minimum version supported by the generator - `12.16` :) I didn't check the first release of `12v` but I guess that there shouldn't be any problems :)

Reply:
Done :)